vignettes/datatable-importing.Rmd

```diff
@@ -148,12 +160,23 @@ my.write = function (x) {
   }
 }
 ```
-When using package as a suggested dependency you should not import it in NAMESPACE file, just mention it in DESCRIPTION file. You also have to use `data.table::` prefix when calling data.table functions because none of them are imported.
+
+When using a package as a suggested dependency, you should not `import` it in the `NAMESPACE` file. Just mention it in the `DESCRIPTION` file. You also _must_ use the `data.table::` prefix when calling `data.table` functions because none of them are imported.
+
+## `data.table` in `Imports:` but nothing exported
```

Comment:
"nothing imported" seems more appropriate, and I would remove `:` from `Imports:`

Reply:
Yes, good